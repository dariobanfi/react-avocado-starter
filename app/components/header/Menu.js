import React from 'react';
import ButtonLink from '../ui/ButtonLink';

const Menu = () => (
  <div>
    <ButtonLink to="/">Home</ButtonLink>
    <ButtonLink to="/info">Info</ButtonLink>
  </div>
);

export default Menu;
