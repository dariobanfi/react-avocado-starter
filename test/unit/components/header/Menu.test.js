import React from 'react';
import { shallow } from 'enzyme';

import Menu from '../../../../app/components/header/Menu';

describe('<Home />', () => {
  test('renders', () => {
    const wrapper = shallow(<Menu />);
    expect(wrapper).toMatchSnapshot();
  });
});
