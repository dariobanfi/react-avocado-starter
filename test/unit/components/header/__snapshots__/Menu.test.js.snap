// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders 1`] = `
<div>
  <Styled(Link)
    to="/"
  >
    Home
  </Styled(Link)>
  <Styled(Link)
    to="/info"
  >
    Info
  </Styled(Link)>
</div>
`;
